cli.js

```diff
@@ -17,7 +18,14 @@ var cli = meow({
 	]
 });
 
-var fn = cli.flags.ipv6 ? 'v6' : 'v4';
+var fn;
+if (Object.keys(cli.flags).length === 0 || cli.flags['4'] || cli.flags.ipv4) {
```

Comment:
`Object.keys(cli.flags).length === 0` is not a good check as it will break if we ever add more flags here.